src/Types/index.tsx

```diff
@@ -249,7 +273,7 @@ export enum InfluxColors {
   Topaz = '#E85B1C',
   Tiger = '#F48D38',
   Pineapple = '#FFB94A',
-  Thunder = '#FFD255',
+  //Thunder = '#FFD255',
```

Comment:
If we're trying to delete this color we should remove it instead of commenting it out.